internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -460,3 +462,28 @@ func isCorrectSvcNameFormat(s string) bool {
 	trailingMatch := trailingPunctRegExp.FindStringSubmatch(s)
 	return len(trailingMatch) == 0
 }
+
+func validateTopicSubscription(ts manifest.TopicSubscription, validTopicARNs []string) error {
+	if err := validatePubSubName(ts.Name); err != nil {
+		return err
+	}
+
+	if err := validateSvcName(ts.Service); err != nil {
+		return err
+	}
+
+	// Check that the topic is included in the list of available topics
+	for _, topicARN := range validTopicARNs {
+		splitArn := strings.Split(topicARN, ":")
```

Comment:
aws-sdk has this [arn package](https://docs.aws.amazon.com/sdk-for-go/api/aws/arn/#ARN) that we can take advantage of!